src/core_ocean/shared/mpas_ocn_diagnostics.F

```diff
@@ -410,8 +410,7 @@ subroutine ocn_diagnostic_solve(dt, statePool, forcingPool, meshPool, diagnostic
 
       deallocate(div_hu,div_huTransport,div_huGMBolus)
 
-      nEdges = nEdgesArray( 2 )
-
+      nEdges = nEdgesArray( 3 )
```

Comment:
@vanroekel FYI, once we used `tangentialVelocity` in the Richardson number, we had to expand this halo to pass the partition test.